ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите внешний диаметр:");
+            Double A = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Введите внутренний диаметр:");
+            Double B = Convert.ToDouble(Console.ReadLine());
+            double C = (Math.PI / 4) * ((A * A) - (B * B));
+            Console.WriteLine("Площадь(S) =" + Math.Round(C, 2));
```

Comment:
В задании не требовалось округлять результат.
Стиль: локальные переменные принято называть, начиная с маленькой буквы. 
```suggestion
            Double a = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Введите внутренний диаметр:");
            Double b = Convert.ToDouble(Console.ReadLine());
            double c = (Math.PI / 4) * ((a * a) - (b * b));
            Console.WriteLine("Площадь(S) =" + c);
```